app/src/main/res/values-night/color_palette.xml

```diff
@@ -19,7 +19,7 @@
   <color name="color_palette_highlighted_background_color">@color/color_def_highlight_blue</color>
   <color name="color_palette_light_description_text_color">@color/color_def_white</color>
   <color name="color_palette_primary_background_color">@color/color_def_oppia_light_black</color>
-  <color name="color_palette_hint_text_color">@color/color_def_accessible_light_grey</color>
```

Comment:
Are you sure this was not getting used anywhere else before removing the previous color?

Reply:
In that case the color for `layout/math_expression_parser_fragment.xml` will get changed and therefore creating an issues. I suggest creating a new color_palette and using that.

Reply:
> In that case the color for `layout/math_expression_parser_fragment.xml` will get changed and therefore creating an issues. I suggest creating a new color_palette and using that.

Done